tools/deviceGen/output/SPhysicalDeviceFeatures_members.h

```diff
@@ -0,0 +1,1943 @@
+
+    // [EXPOSE AS A LIMIT] ROADMAP2022 wants them, but device support low
+    // Vulkan 1.3 requires but we make concessions for MoltenVK
+    // bool vulkanMemoryModelAvailabilityVisibilityChains = false;
+
+    // Vulkan 1.2 Core or VK_EXT_shader_viewport_index_layer
+    // [EXPOSE AS A LIMIT]
+    // ALIAS: VK_EXT_shader_viewport_index_layer
+    // bool shaderOutputViewportIndex = false;
+    // ALIAS: VK_EXT_shader_viewport_index_layer
+    // bool shaderOutputLayer = false;
+
+    // [REQUIRE]
+    // ubiquitous device support
+    // bool subgroupBroadcastDynamicId = true;
+
+    // Vulkan 1.3 Core
+
+    // This feature adds stricter requirements for how out of bounds reads from images are handled.
+    // Rather than returning undefined values,
+    // most out of bounds reads return R, G, and B values of zero and alpha values of either zero or one.
+    // Components not present in the image format may be set to zero
+    // or to values based on the format as described in Conversion to RGBA in vulkan specification.
+    // widely supported but has performance overhead, so remains an optional feature to enable
+    // or VK_EXT_image_robustness
+    bool robustImageAccess = false;
+
+    // [REQUIRE][DO NOT EXPOSE] VK_EXT_inline_uniform_block EVIL regressive step back into OpenGL/Dx10 times? Or an intermediate step between PC and UBO?
+    // Vulkan 1.3, Nabla Core Profile:
+    // bool inlineUniformBlock = false;
+    // ROADMAP 2022, Nabla Core Profile:
+    // bool descriptorBindingInlineUniformBlockUpdateAfterBind = false;
+
+    // [REQUIRE]
+    // Vulkan 1.3 non-optional and Nabla Core Profile but TODO: need impl
+    // or VK_EXT_pipeline_creation_cache_control
+    // bool pipelineCreationCacheControl = true;
+
+    // [REQUIRE]
+    // [DO NOT EXPOSE] ever we have our own mechanism, unless we can somehow get the data out of `VkObject`?
+    // or VK_EXT_private_data
+    // bool privateData = false;
+
+    // Vulkan 1.3 non-optional requires but poor support
+    // [EXPOSE AS LIMIT] Vulkan 1.3 non-optional requires but poor support
+    // or VK_EXT_shader_demote_to_helper_invocation
+    // bool shaderDemoteToHelperInvocation = false;
+    // or VK_KHR_shader_terminate_invocation
+    // bool shaderTerminateInvocation = false;
+
+    // [REQUIRE]Nabla Core Profile, Vulkan 1.3 or VK_EXT_subgroup_size_control
+    // bool subgroupSizeControl = true;
+    // bool computeFullSubgroups = true;
+
+    // [REQUIRE]
+    // REQUIRE
+    // or VK_KHR_synchronization2
+    // bool synchronization2 = true;
+
+    // [DO NOT EXPOSE] Doesn't make a difference, just shortcut from Querying support from PhysicalDevice
+    // or VK_EXT_texture_compression_astc_hdr
+    // bool textureCompressionASTC_HDR;
+
+    // [EXPOSE AS LIMIT] Vulkan 1.3 non-optional requires but poor support
+    // or VK_KHR_zero_initialize_workgroup_memory
+    // bool shaderZeroInitializeWorkgroupMemory = false;
+
+    // [REQUIRE]
+    // [DO NOT EXPOSE] EVIL
```

Comment:
Require and DO NOT expose at the same time, wtf?

yeah you really need to disambiguate between REQUIRE and DISABLE